redis_rate_limit/__init__.py

```diff
@@ -88,7 +92,6 @@ def get_usage(self):
         """
         Returns actual resource usage by client. Note that it could be greater
         than the maximum number of requests set.
-
         :return: integer: current usage
         """
         return int(self._redis.get(self._rate_limit_key) or 0)
```

Comment:
I think the check should be done here.

```suggestion
        if self.ignored_clients and self.client in self.ignored_clients:
            return 0

        return int(self._redis.get(self._rate_limit_key) or 0)
```

There are two reasons for it:

- adding a client to the list has always an instant bypass effect
- removing a client from the list automatically enforces the limits again

The reason is to avoid edge cases like this:

- client reaches max requests limit
- client is added to the ignored clients list
- client will still need to wait for expire time before being allowed to make requests again

Or this other one:

- client is added to ignored list by mistake and overflows max number of requests
- client is removed from the list
- there's no back-off

What do you think, @italorossi?